pkg/sqlqueries/SQL/mto/milestone/create_milestone_solution_link.sql

```diff
@@ -0,0 +1,32 @@
+WITH QUERIED_IDS AS (
+    /*Translate the input to a table */
+    SELECT
+        id,
+        milestone_id,
+        solution_id,
+        created_by,
+        created_dts,
+        modified_by,
+        modified_dts
+    FROM
+        JSON_TO_RECORDSET(:paramTableJSON) --noqa
+)
+
+INSERT INTO mto_milestone_solution_link (
+    id,
+    milestone_id,
+    solution_id,
+    created_by,
+    created_dts,
+    modified_by,
+    modified_dts
+)
+SELECT
+    qIDs.id,
+    qIDs.milestone_id,
+    qIDs.solution_id,
+    qIDs.created_by,
+    qIDs.created_dts,
+    qIDs.modified_by,
+    qIDs.modified_dts
+FROM QUERIED_IDS AS qIDs;
```

Comment:
@StevenWadeOddball This is a distinct approach in an attempt to batch creation using loaders. It may be over-engineering. I can rework this to a simple query if you think it's appropriate.

Reply:
Thanks Tom! 

This is an Interesting idea, I'm not sure if there are any pitfalls with it but I can't think of it now! I think it would be interesting to proceed with it and see what happens! This wouldn't be a dataloader correct? Just a direct store call? (I think that's good, a dataloader might have other repercussions)

Were you thinking that you would do an ON CONFLICT / MERGE statement? If you did that, you wouldn't have to worry if the link already exists, you could just not error. 

That does bring up considerations that need to be handled around the generating the ids. You might consider getting the ID with gen_random_uuid or similar in the SQL instead of app code (since you wouldn't know if the link exists yet or not).

Reply:
I wound up simplifying this logic due to timeboxing constraints